src/page/CheckList/components/CheckListItemCard.tsx

```diff
@@ -91,6 +91,13 @@ const getStyles = (theme: GrafanaTheme2) => ({
     backgroundColor: theme.colors.background.secondary,
     borderRadius: '2px',
   }),
+  labelsContainer: css({
+    display: 'flex',
+    flexWrap: 'wrap',
+    flexGrow: 1,
+    maxWidth: '90%',
```

Comment:
Why not 100%? Looks way better.
![image](https://github.com/user-attachments/assets/f093967f-1123-47f9-a646-d0140c48c29e)

vs 90%
![image](https://github.com/user-attachments/assets/b24961a2-533c-49f9-ac1a-fe617e94a0c8)